source/switching_to_live/set_up_a_live_stripe_account/index.html.md.erb

```diff
@@ -7,6 +7,8 @@ weight: 161
 
 This is part of [going live](/switching_to_live/#go-live).
 
+Stripe is GOV.UK Pay's payment service provider (PSP).
```

Comment:
"centrally procured" PSP? maybe match langauge to the product page?

Reply:
I think that level of detail isn't necessary for the docs - the user of the docs doesn't need that information